src/components/Table/hoc/withTableActions/withTableActions.tsx

```diff
@@ -54,7 +54,7 @@ export type TableActionConfig<I> = TableAction<I> | TableActionGroup<I>;
 
 export interface WithTableActionsProps<I> {
     getRowActions: (item: I, index: number) => TableActionConfig<I>[];
-    rowActionsSize?: ButtonSize;
+    rowActionsSize?: Omit<ButtonSize, 'xs'>;
```

Comment:
Prop is used for Button and Menu components at the same time. But Menu doesn't support xs size.